src/framework/audio/worker/internal/mixer.cpp

```diff
@@ -275,6 +284,7 @@ void Mixer::processTrackChannels(size_t outBufferSize, size_t samplesPerChannel,
     bool filterTracks = m_isIdle && !m_tracksToProcessWhenIdle.empty();
 
     if (useMultithreading()) {
+#ifdef MUSE_THREADS_SUPPORT
```

Comment:
useMultithreading() already returns false in this case (I just don't like these ifdef blocks)

Reply:
we need to remove the code itself, it won't compile, or rather won't link. since there are no functions for working with threads